arangodb-net-standard/GraphApi/DeleteEdgeDefinitionResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Net;
+
+namespace ArangoDBNetStandard.GraphApi
+{
+    public class DeleteEdgeDefinitionResponse
+    {
+        public HttpStatusCode Code { get; set; }
+
+        public bool Error { get; set; }
+
+        public GetGraphsGraph Graph { get; set; }
```

Comment:
Can we use a specific model here? `DELETE/_api/gharial/{graph}/edge/{definition}` returns a name in the model but `GetGraphsGraph` does not contain a name.

I will create a new issue to refactor the graph models. From what I have observed, all the endpoint that perform modification (management) of the graph definition will return the modified graph.
https://www.arangodb.com/docs/stable/http/gharial-management.html
We could create a `ModifiedGraph` model that is used across those endpoints.